src/cluster/replication.cc

```diff
@@ -921,6 +921,9 @@ rocksdb::Status ReplicationThread::ParseWriteBatch(const std::string &batch_stri
       GetFixed64(&entry_id, &id.seq);
       srv_->OnEntryAddedToStream(ikey.GetNamespace().ToString(), ikey.GetKey().ToString(), id);
       break;
+    }
+    default:
```

Comment:
I don't know whether C++ can resolve exhaustive `case` and we just write `case kBatchTypeNone:` here.

Reply:
I think both of them is OK here.

Reply:
Yep. I'd prefer explicit than implicit but it's not a blocker from merging :)

Reply:
done